src/Automation/Interfaces/IScanner.cs

```diff
@@ -39,10 +60,28 @@ public interface IScanner
         /// All exceptions are wrapped in <see cref="AxeWindowsAutomationException"/>.
         /// If the exception was not thrown by AxeWindows automation, the <see cref="Exception.InnerException"/> property
         /// will contain the exception.
+        /// Calling this function is equivalent to passing false to <see cref="IScanner.Scan(string, bool)"/> and passing false for <c>enableMultipleWindows</c>.
         /// </remarks>
         /// <returns>Information about the scan and any issues detected</returns>
         /// <exception cref="AxeWindowsAutomationException"/>
         ScanResults Scan(string scanId);
 
+        /// <summary>
+        /// Run AxeWindows automated tests on all windows or the first window found.
+        /// </summary>
+        /// <remarks>
+        /// If a value was provided in <see cref="Config.OutputDirectory"/>,
+        /// output files may be written to the specified directory.
+        /// (Note: no output files will be written if no errors were found.)
+        /// An exception may be thrown if the value of <see cref="Config.ProcessId"/> is invalid
+        /// or if the directory provided in <see cref="Config.OutputDirectory"/> cannot be created or accessed.
+        /// All exceptions are wrapped in <see cref="AxeWindowsAutomationException"/>.
+        /// If the exception was not thrown by AxeWindows automation, the <see cref="Exception.InnerException"/> property
+        /// will contain the exception.
+        /// If <paramref name="enableMultipleWindows"/> is set to true, all windows will be scanned. If it is set to false, only the first window will be scanned.
+        /// </remarks>
+        /// <returns>Information about the scan and any issues detected</returns>
+        /// <exception cref="AxeWindowsAutomationException"/>
+        IReadOnlyCollection<ScanResults> Scan(string scanId, bool enableMultipleWindows);
```

Comment:
The original logic for making `scanId` a parameter to `IScanner.Scan` was that someone might create a single `IScanner`, then trigger multiple scans using that object. I think we'd be hard pressed to argue that the same test app would trigger some scans with multiple roots enabled and other scans without. For that reason, I don't think we want to have this mehod